pom.xml

```diff
@@ -241,4 +245,17 @@
         </plugin>
       </plugins>
   </reporting>
+  <profiles>
+    <profile>
+      <id>java12</id>
+      <activation>
+        <jdk>[12,)</jdk>
+      </activation>
+      <properties>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <maven.compiler.release>7</maven.compiler.release>
+      </properties>
+    </profile>
+  </profiles>
```

Comment:
Nice. I will most likely just delete it once I can release and use `mojo-parent:50` which defines JDK7 as the minimum but this is perfect in the meantime.